holoviews/tests/plotting/bokeh/test_rasterplot.py

```diff
@@ -150,3 +151,78 @@ def test_rgb_invert_yaxis(self):
             self.assertEqual(cdata['y'], [-0.5])
         else:
             self.assertEqual(cdata['y'], [0.5])
+
+    def test_image_stack(self):
+        x = np.arange(0, 3)
+        y = np.arange(5, 8)
+        a = np.array([[np.nan, np.nan, 1], [np.nan] * 3, [np.nan] * 3])
+        b = np.array([[np.nan] * 3, [1, 1, np.nan], [np.nan] * 3])
+        c = np.array([[np.nan] * 3, [np.nan] * 3, [1, 1, 1]])
+
+        img_stack = ImageStack((x, y, a, b, c), kdims=["x", "y"], vdims=["a", "b", "c"])
+        plot = bokeh_renderer.get_plot(img_stack)
+        source = plot.handles['source']
+        self.assertEqual(source.data['image'][0][0], a.T)
+        self.assertEqual(source.data['image'][0][1], b.T)
+        self.assertEqual(source.data['image'][0][2], c.T)
+        self.assertEqual(source.data['x'][0], -0.5)
+        self.assertEqual(source.data['y'][0], 4.5)
+        self.assertEqual(source.data['dw'][0], 3)
+        self.assertEqual(source.data['dh'][0], 3)
+        assert isinstance(plot, ImageStackPlot)
+
+    def test_image_stack_invert_xaxis(self):
+        x = np.arange(0, 3)
+        y = np.arange(5, 8)
+        a = np.array([[np.nan, np.nan, 1], [np.nan] * 3, [np.nan] * 3])
+        b = np.array([[np.nan] * 3, [1, 1, np.nan], [np.nan] * 3])
+        c = np.array([[np.nan] * 3, [np.nan] * 3, [1, 1, 1]])
+
+        img_stack = ImageStack((x, y, a, b, c), kdims=["x", "y"], vdims=["a", "b", "c"])
+        plot = bokeh_renderer.get_plot(img_stack.opts(invert_xaxis=True))
+        source = plot.handles['source']
+        self.assertEqual(source.data['image'][0][0], a.T)
```

Comment:
Why isn't this flipped?
```suggestion
        self.assertEqual(source.data['image'][0][0], a.T[::-1])
```

Reply:
Isn't axis inversion handled by flipping the x and y values, rather than flipping the actual data?